rbi/core/symbol.rbi

```diff
@@ -318,4 +318,8 @@ class Symbol < Object
   # an object. As *sym* is already a symbol, `self` is returned in this case.
   sig {returns(T.self_type)}
   def to_sym(); end
+
+  # Returns the name or string corresponding to *sym*. Unlike `to_s`, the returned string is frozen.
```

Comment:
I manually copied this over from the Ruby 3 docs, is there a better approach I should use?